mlx/traceability.py

```diff
@@ -718,6 +754,32 @@ def process_item_nodes(app, doctree, fromdocname):
 
         node.replace_self(new_node)
 
+    # We process the item-link items. These are stored in a dictionary, so they
```

Comment:
no need to store in seperate dict here. Just parse the object, your `node` variable and use `env.traceability_collection` to add the relations using `add_relation`. Example in line 171 of this file.